securedrop/source_templates/footer.html

```diff
@@ -3,7 +3,7 @@
     {{ gettext('Powered by') }} <b>SecureDrop {{ version }}</b>.
   </p>
   <p id="footer-advisory">
-    {{ gettext('Please note: Sharing sensitive documents may put you at risk, even when using Tor and SecureDrop.') }}
+    {{ gettext('Please note: Sharing sensitive files may put you at risk, even when using Tor and SecureDrop.') }}
```

Comment:
I might be tempted to make this "sensitive information", since messages without files could carry the same risk, but "submissions" doesn't make sense in this context.